tools/pagerduty-migrator/migrator/resources/integrations.py

```diff
@@ -52,7 +53,14 @@ def migrate_integration(integration: dict, escalation_policies: list[dict]) -> N
 def create_integration(
     name: str, integration_type: str, escalation_chain_id: str
 ) -> None:
-    payload = {"name": name, "type": integration_type, "team_id": None}
+
+    DEFAULT_INTEGRATION_TYPE = ["alertmanager", "grafana", "webhook", "kapacitor", "elastalert", "zabbix"]
```

Comment:
Please see [PAGERDUTY_TO_ONCALL_VENDOR_MAP](https://github.com/grafana/oncall/blob/dev/tools/pagerduty-migrator/migrator/config.py#L26), I think here it's better to use `PAGERDUTY_TO_ONCALL_VENDOR_MAP.values()` instead of introducing a new variable.